core/src/test/scala/com/vladkopanev/zio/saga/SagaSpec.scala

```diff
@@ -85,6 +88,39 @@ object SagaSpec
             } yield assert(log)(equalTo(Vector("flight canceled", "hotel canceled")))
           }
         ),
+        suite("Saga#zipWith")(
+          test("should successfully run two Sagas in sequence") {
+          for {
+            startTime <- currentTime(TimeUnit.MILLISECONDS).provide(Clock.live)
+            _ <- (sleep(1000.millis) *> bookFlight compensate cancelFlight)
+              .zipWith(sleep(1000.millis) *> bookHotel compensate cancelHotel)((_, _) => ())
+              .transact
+            endTime <- currentTime(TimeUnit.MILLISECONDS).provide(Clock.live)
+          } yield assertTrue(endTime - startTime >= 2000L)
```

Comment:
actually, this test is designed to check parallel behaviour which is not the case with `zipWith` function, so we shouldn't actually check the timing here, let's better just check results of both actions, that they are expected

Reply:
Agree. Let me re-write it. In that scenario its not required to extract `measure` or similar in SpecUtil